x-pack/plugins/alerts/server/saved_objects/mappings.json

```diff
@@ -71,6 +71,9 @@
       "throttle": {
         "type": "keyword"
       },
+      "notifyOnStateChange": {
```

Comment:
Are we going to want a migration to set this to `false` by default?  Presumably when we get back last-version documents, the referenced property will be `undefined` and thus falsey, but seems safer to add it in explicitly, lest we find some code expecting a boolean and finding an `undefined`.  @mikecote ?

Reply:
That's correct, if we want explicit `true/false` values at all times, we'll need a migration for this. I'm personally a fan of explicit over implicit, so it's a +1 from me.

Reply:
Migration added.